### ISSUE TITLE
Convert to bikeshed

### ISSUE THREAD

**Issue by zcorpan** [whatwg/html-differences#2]
cc @annevk

**Comment by annevk** [comment]
See the last three commits here for guidance: https://github.com/whatwg/encoding/commits. Or look at Fullscreen.

**Comment by annevk** [comment]
Adding IDs to headings, then renaming resources, than doing the rest, in three commits, is a rough summary of the process and has worked well thus far.

**Comment by annevk** [comment]
Given the lack of updates to this document, maybe we should just can it altogether?

**Comment by annevk** [comment]
We could maybe reach out to the mailing list and the blog first?

**Comment by zcorpan** [comment]
Yeah... the target audience for this document is probably small enough today that it isn't worth the cost to maintain. Unless someone is interested in taking over as editor we should probably mark it as being a stale resource somehow.

**Comment by annevk** [comment]
Is this the only document that still depends on whatwg/xref? I suspect we'll end up breaking some of that soon in HTML if we change how fragment-links works for instance. Is that a problem?

**Comment by zcorpan** [comment]
I don't think so, particularly. If we want to maintain this we should finish conversion to bikeshed. If not, we'll still make sure existing links to HTML continue to work, right?

**Comment by annevk** [comment]
Yeah, it would just be a problem if you wanted to generate a new html-generated.json.

**Comment by annevk** [comment]
Though really I'd like to see whatwg/xref gone to avoid confusing folks. I just added a README that marks it obsolete, hopefully that helps.
